sui/src/unit_tests/shell_tests.rs

```diff
@@ -216,3 +217,19 @@ fn test_completer_with_cache() {
         .collect::<Vec<_>>();
     assert_eq!(vec!["--gas"], candidates);
 }
+
+#[test]
+fn test_split_line() {
```

Comment:
I think an easy test would be to run this through the [BLNS](https://github.com/minimaxir/big-list-of-naughty-strings).

Reply:
Nice, I will try to add that after DevNet